internal/main

```diff
@@ -205,8 +205,8 @@ function install_check_old_files {
     SIGNATURE_FILES=(
 	Jenkinsfile
 	Makefile
+	requirements.txt
 	docs/Makefile
-	tox.ini
```

Comment:
Won't some projects have the tox.ini?

Reply:
This is just the list of files to look for to see if we think we're using the legacy toolkit. The full list of files to remove comes later.